dotcom-rendering/src/web/components/Footer.tsx

```diff
@@ -236,16 +237,31 @@ const FooterLinks = ({
 
 const year = new Date().getFullYear();
 
+// Hard coding the values here is not ideal - ideally, they would sourced from the
+// newsletters API by frontend and included in the request.
```

Comment:
```suggestion
```

I think it's okay to have these values hard coded here. We do this in a lot of other places so it's an established pattern

Reply:
I agree with Oliver that updating `DotcomRenderingDataModel` in `frontend` to pass these is not a path you need to go down just yet.

I think the comment is helpful, though, as it can help us weigh the number of use-cases for these kind of uses.